tools/GraphClust/GSPAN/structure_to_gspan.xml

```diff
@@ -7,29 +7,28 @@
     </stdio>
     <command>
     <![CDATA[
-      structure_to_gspan.pl
+        mkdir ./temp/ ./GSPAN_Outputs/ &&
+        structure_to_gspan.pl
             --input-file '$dataFile'
-            --input-format 
-            $i_abstr
-            $i_stacks
+            --input-format 'rnafold'
+            $abstr
+            $stack
             $seq_graph_t
             -group $group
-
-
+            -tmp ./temp
+            -o ./GSPAN_Outputs/
     ]]>
     </command>
     <inputs>
         <param type="data" name="dataFile" format="data" label="Sequence and Structure input file in the prediction tool format" />
-        <param name="inputFormat" type="text" value="rnafold"
```

Comment:
@bgruening The interface may accept other output (e.g. RNAshapes in the future). Though currently only on format is supported.